prime-angular-frontend/src/app/modules/adjudication/shared/services/adjudication-resource.service.ts

```diff
@@ -334,6 +334,22 @@ export class AdjudicationResource {
       );
   }
 
+  public changeAgreementType(enrolleeId: number, note: string, agreementType: number): Observable<EnrolleeNote> {
+    const payload = { note, agreementType };
+    return this.apiResource.put(`enrollees/${enrolleeId}/status-actions/change-toa`, payload)
+      .pipe(
+        map((response: ApiHttpResponse<EnrolleeNote>) => response.result),
+        tap(() => {
+          this.toastService.openSuccessToast(`Agreement type changed.`);
+        }),
+        catchError((error: any) => {
+          this.toastService.openErrorToast('Agreement could not be changed');
```

Comment:
"Agreement type could not be changed" more accurate

Reply:
Updated